hazelcast/src/main/java/com/hazelcast/scheduledexecutor/TaskHelper.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.scheduledexecutor;
+
+import com.hazelcast.scheduledexecutor.impl.NamedTaskDecorator;
+import com.hazelcast.spi.annotation.Beta;
+
+import java.util.concurrent.Callable;
+
+/**
+ * A helper class with utilities to act upon {@link Runnable} and/or {@link Callable} tasks.
+ */
+@Beta
+public final class TaskHelper {
```

Comment:
Why don't we call this class TaskUtils? To be in line with the other utility classes.